perf/lua/uri_escape_unescape.lua

```diff
@@ -5,6 +5,13 @@ local uri = require("uri")
 local clock = require("clock")
```

Comment:
commit message to "perf: standardize an output of uri_escape_unescape benchmark "

Reply:
In that case, the commit title is more than 50 symbols, so I shorten it to this one.